templates/google-services.json

```diff
@@ -15,7 +15,7 @@
         }
       },
       "oauth_client": [],
-      "api_key": [],
+      "api_key": [{ "current_key": "" }],
```

Comment:
Required to update from `com.google.gms:google-services:2.0.0` to `com.google.gms:google-services:3.0.0`